qa/tasks/cephfs/test_nfs.py

```diff
@@ -141,15 +141,31 @@ def _test_idempotency(self, cmd_func, cmd_args):
         if ret != 0:
             self.fail("Idempotency test failed")
 
-    def _test_create_cluster(self):
+    def _test_create_cluster(self, _sleep=3, _tries=3):
```

Comment:
Does this needs to configurable. Just use sane values.

Reply:
okay

Reply:
done